index.js

```diff
@@ -59,13 +60,18 @@ module.exports = {
   },
 
   _getComparisonPlugin() {
-    const trueEmberVersion = this.emberVersion.match(/\d+\.\d+\.\d+/)[0];
+    const trueEmberVersion = extractTrueVersion(this.emberVersion);
 
-    return [require.resolve('./comparision-plugin.js'), { emberVersion: trueEmberVersion }];
+    return [require.resolve('./comparision-plugin.js'), { emberVersion: trueEmberVersion, root: this.project.root }];
```

Comment:
This is a little bit of a sticky wicket here because sometimes you want `this.parent.root` (for deps that are allowed to be duplicated) and othertimes you want `this.project.root` (for those that are only deps of the top level project).

I take it with this choice, you are thinking that top level deps are the most important?

Reply:
Yes. I made an assumption here that addons must follow the highlander rule for dependencies available within the app.

Reply:
OK, seems fine for now, we can address with either separate methods (e.g. `parentDepGte`) if it ever is an issue.